src/app/child-dev-project/children/model/childSchoolRelation.ts

```diff
@@ -3,13 +3,20 @@ import { DatabaseEntity } from "../../../core/entity/database-entity.decorator";
 import { DatabaseField } from "../../../core/entity/database-field.decorator";
 import moment from "moment";
 import { School } from "../../schools/model/school";
+import { Child } from "./child";
 
 /**
  * Record of a school year that a Child attended a certain class in a School.
  */
 @DatabaseEntity("ChildSchoolRelation")
 export class ChildSchoolRelation extends Entity {
-  @DatabaseField() childId: string;
+  @DatabaseField({
+    label: $localize`:Label for the child of a relation:Child`,
```

Comment:
A german translation has to be provided for this

Reply:
See our guide for [building localizable components](https://aam-digital.github.io/ndb-core/documentation/additional-documentation/how-to-guides/build-localizable-components.html)